scripts/launch-nodes/README.md

```diff
@@ -67,24 +69,47 @@ host = "farmer.example.com"
 user = "username"
 ssh_key = "/path/to/private/key"
 type = "farmer"
+
+[timekeeper]
+host = "timekeeper.example.com"
+user = "username"
+ssh_key = "/path/to/private/key"
+type = "timekeeper"
+
 ```
 
 - **`bootstrap_node`:** This section defines the Bootstrap node.
 - **`farmer_rpc_nodes`:** This section contains the RPC and Farmer nodes. The `type` field specifies whether the node is an RPC node or a Farmer node.
+- **`timekeeper`:** This section Defines the Timekeeper node..
 
 ### Step 4: Running the Script
 
 Once the configuration file is ready, make the python script executable and run the Python script with the following command:
 
 ```bash
 chmod +x manage_subspace.py
-python manage_subspace.py --config nodes.toml --release_version gemini-3h-2024-sep-17 --subspace_dir /home/ubuntu/subspace/subspace --pot_external_entropy random_value
+python manage_subspace.py --config nodes.toml --release_version gemini-3h-2024-sep-17 --subspace_dir /home/ubuntu/subspace/subspace \
+--pot_external_entropy random_value --network gemini-3h --plot-size 10G --cache-percentage 15
+
+# prune images
+python manage_subspace.py --config nodes.toml --release_version gemini-3h-2024-sep-17 --subspace_dir /home/ubuntu/subspace/subspace --prune
```

Comment:
Would this prune the release specified in the `release_version` or it will prune all other releases except the one specified?